post-scan-actions/aws-python-conformity-custom-check/template.yml

```diff
@@ -21,12 +21,18 @@ Parameters:
     Description: ARN of the File Storage SNS Topic to subscribe this function.
   ConformityRegion:
     Type: String
-    Description: Region where you conformity account is hosted. (us-west-2 for Cloud One customers)
-    Default: us-west-2
+    Description: Region where you conformity account is hosted. (us-1 for Cloud One customers)
```

Comment:
Since all regions are available for C1 customers now, is `us-1 for Cloud One customers` an outdated messages?

Reply:
I have deleted this message.